app/jobs/upvs/fetch_public_authority_edesks_list_job.rb

```diff
@@ -46,6 +46,8 @@ def check_row_attributes(attributes)
     if name !~ /TEST/i
       raise "#{uri} does not match #{cin}" if uri !~ /ico:\/\/sk\/(0*)#{cin}(_\d+)?/
     end
+
+    raise "Incorrect encoding" if name =~ /.*\\u.*/
```

Comment:
Ked to zlyha vzdy to vyzera takto? Rozmyslam ci na to netreba ist naopak, ze hladat ci tam su len povolene znaky.

Reply:
To asi ja neviem posudit. Povedz ty alebo @michal-rohacek a mozem prerobit.

Reply:
Zlyhalo to len par krat a doteraz vsetky chyby mali tento format - [tuto je tusim podobny pripad](https://sluzbyslovenskodigital.atlassian.net/browse/GB-136)

Kazdopadne statne data dost nespolahlive, teda nejaky character whitelist moze byt dobry napad - len rozmyslam, ci vieme presne definovat tu mnozinu znakov (mozno je nejaka libka/specifikacia na to), pripadne performance impact - aj ked tych entit je myslim pod 10k, cize to asi nebude problem.

Reply:
Sposobov ako to pokazit je asi nekonecno, preto skor uvazujem nad whitelisting. Ale neviem velmi vymysliet pravidlo, ktore by bolo nepriestrelne. Cize skusme to takto a uvidime co sa stane.